charts/nuts-node/templates/nuts-data-pv.yaml

```diff
@@ -8,22 +9,26 @@ metadata:
     "helm.sh/resource-policy": keep
 spec:
   persistentVolumeReclaimPolicy: Retain
-  storageClassName: ""
+  storageClassName: {{ .Values.nuts.data.persistedVolume.storageClassName | default "" }}
   capacity:
     storage: {{ .Values.nuts.data.persistedVolume.capacity | default "100Mi" }}
   accessModes: {{ required "Please define `nuts.data.persistedVolume.accessModes` in `values.yaml`" .Values.nuts.data.persistedVolume.accessModes }}
   hostPath:
-    path: {{ .Values.nuts.data.persistedVolume.mountPath | default "/opt/nuts/data" }}
+    path: {{ .Values.nuts.data.persistedVolume.mountPath  }}
```

Comment:
Is there a specific reason why the default is removed?

Reply:
@rolandgroen ?